packages/components/src/components/omnibox/actions/Filter.ts

```diff
@@ -164,10 +165,12 @@ export class FilterAction implements Action {
     optionalLabel = 'columns';
     getColumns: (all?: boolean) => List<QueryColumn>;
     urlPrefix: string;
+    getFilterDisplayValue: (columnName: string, rawValue: string) => string;
 
-    constructor(urlPrefix: string, getColumns: () => List<QueryColumn>) {
+    constructor(urlPrefix: string, getColumns: () => List<QueryColumn>, getQueryInfo?: () => QueryInfo, getFilterDisplayValue?: (columnName: string, rawValue: string) => string) {
```

Comment:
@XingY  Why the addition of the `getQueryInfo` param here? I don't see it used anywhere in this PR.

Reply:
Yeah, this is confusing. I've added a comment to explain why this param is needed. The 3rd param is already claimed by View action.

Perhaps it would be cleaner if we had defined an interface for Action's constructor param and use an object instead of any array here instead.